src/Migration/Internal/MigrationBuilder.cs

```diff
@@ -24,11 +24,19 @@ public class MigrationBuilder : IMigrationBuilder
         /// <exception cref="T:System.ArgumentException">stepName</exception>
         public virtual IMigrationBuilder Step(string stepName, Action action)
         {
+#if NET45
+            return Step(stepName, async () =>
+            {
+                action();
+                await Task.FromResult(0);
```

Comment:
It might be nicer to conditionally add CompletedTask as an extension method on `Task`.

Reply:
It definitely would, but AFAIK we can't create extension`properties` in dotnet